components/layout_2020/flow/mod.rs

```diff
@@ -1632,14 +1629,14 @@ impl PlacementState {
                     self.current_margin
                         .adjoin_assign(&fragment_block_margins.end);
                 } else {
-                    self.current_block_direction_position +=
-                        self.current_margin.solve() + fragment_block_size.into();
+                    (self.current_margin.solve() + fragment_block_size.into());
```

Comment:
Is this addition saved anywhere? The `self.current_block_direction_position +=` from before seems to be missing, you may need to revert this change.